-------------------------------------------------------------------------------
Test set: initial.testdatadriven.DataFileTest
-------------------------------------------------------------------------------
Tests run: 10, Failures: 0, Errors: 10, Skipped: 0, Time elapsed: 0.042 sec <<< FAILURE!
testA ReTest[1](initial.testdatadriven.DataFileTest)  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[2](initial.testdatadriven.DataFileTest)  Time elapsed: 0.002 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[3](initial.testdatadriven.DataFileTest)  Time elapsed: 0.003 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[4](initial.testdatadriven.DataFileTest)  Time elapsed: 0.003 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[5](initial.testdatadriven.DataFileTest)  Time elapsed: 0.004 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[6](initial.testdatadriven.DataFileTest)  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[7](initial.testdatadriven.DataFileTest)  Time elapsed: 0.02 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[8](initial.testdatadriven.DataFileTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[9](initial.testdatadriven.DataFileTest)  Time elapsed: 0.006 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
testA ReTest[10](initial.testdatadriven.DataFileTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.ArrayIndexOutOfBoundsException: 2
	at org.retest.datafile.TestDataFiles.serialize(TestDataFiles.java:40)
	at initial.datafile.NewFileTest.saveBodyItem(NewFileTest.java:50)
	at initial.datafile.NewFileTest.save(NewFileTest.java:40)
	at org.retest.datafile.SuccessTestDataFiles.save(SuccessTestDataFiles.java:33)
	at org.retest.ReTestStatement$1$1.evaluate(ReTestStatement.java:85)
	at org.retest.ReTestStatement.runWithCompleteAssignment(ReTestStatement.java:115)
	at org.retest.ReTestStatement.evaluate(ReTestStatement.java:57)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.retest.ReTestRunner.runRepeatedly(ReTestRunner.java:179)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:154)
	at org.retest.ReTestRunner.runChild(ReTestRunner.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:53)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
